src/doc/release/v1_10_0.md

```diff
@@ -0,0 +1,16 @@
+iCub 1.10.0 (UNRELEASED)  Release Notes
+========================
+
+New Features
+------------
+
+### Libraries
+
+#### `skinDynLib`
+Add `linkName` and `frameName` attributes to `iCub::skinDynLib::dynContact` and `iCub::skinDynLib::skinContact`, to simplify 
```

Comment:
There are several reasons why I prefer to break paragraph in raw Markdown, even if this has no effect on the rendered document: 
* Small lines simplify the usage of git, as merging, diffing, conflicts, etc etc is all managed at line (in the raw document) level
*  One of the main features of Markdown is to be quite readable even when it not rendered, and (at least in my case) I typically edit Markdown parallel with source code, meaning that no soft wrap is enabled in the editor.